### PR TITLE
build middleware echoapp on ci

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

build middleware echoapp on ci

## Issue reference

N/A

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] The sample code compiles correctly
* [ ] You've tested new builds of the sample if you changed sample code
* [x] You've updated the sample's README if necessary

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
             echo "##[set-env name=SAMPLE_REGISTRY;]$SAMPLE_REGISTRY"
           fi
   
-          SAMPLE_LIST=(2.hello-kubernetes 3.distributed-calculator 4.pub-sub 5.bindings)
+          SAMPLE_LIST=(2.hello-kubernetes 3.distributed-calculator 4.pub-sub 5.bindings 7.middleware)
```
Comment (yaron2) [inline .github/workflows/build.yml]: should be `6.middleware`

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -44,7 +44,7 @@ jobs:
       - name: publish image to dockerhub
         if: github.event_name != 'pull_request'
         run: |
-          SAMPLE_LIST=(2.hello-kubernetes 3.distributed-calculator 4.pub-sub 5.bindings)
+          SAMPLE_LIST=(2.hello-kubernetes 3.distributed-calculator 4.pub-sub 5.bindings 7.middleware)
```
Comment (yaron2) [inline .github/workflows/build.yml]: should be `6.middleware`

File: 7.middleware/README.md [hunk 3]
```diff
@@ -26,15 +26,15 @@ helm install --name my-release stable/nginx-ingress
 1. Clone the sample repo, then navigate to the middleware sample:
 ```bash
 git clone https://github.com/dapr/samples.git
-cd samples/7.middleware
+cd samples/7.middleware/echoapp
```
Comment (yaron2) [inline 7.middleware/README.md]: should be `6.middleware`

Comment (youngbupark) [reply 7.middleware/README.md]: @yaron2 keda sample is sample 6

**Comment by yaron2** [comment]
Ignore my comments, I missed the KEDA samples.

File: .github/workflows/build.yml (hunk 2 above)
Comment (youngbupark) [reply .github/workflows/build.yml]: keda is sample 6

File: .github/workflows/build.yml (hunk 1 above)
Comment (youngbupark) [reply .github/workflows/build.yml]: keda!

**Pull request merged**